recipes/meme/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+MEME_ETC_DIR=${PREFIX}/etc
+cpanm YAML
+cpanm HTML::PullParser
```

Comment:
I know :( but this would be the proper way.
I'm not stopping this PR, just letting you know what the best-practise is in bioconda.